httpserver/bin/static_file_server.dart

```diff
@@ -19,7 +19,8 @@ void main() {
     staticFiles.serveFile(new File(indexUri.toFilePath()), request);
   };
 
-  HttpServer.bind(InternetAddress.LOOPBACK_IP_V4, 4048).then((server) {
-    server.listen(staticFiles.serveRequest);
-  });
-}
\ No newline at end of file
+  var server = await HttpServer
+      .bind(InternetAddress.LOOPBACK_IP_V4, 4048);
+  server.listen(staticFiles.serveRequest);
```

Comment:
I'd recommend to use `.forEach` instead of `.listen` when you are not adding any error handlers.